frontend/src/app/render_wasm/api.cljs

```diff
@@ -469,6 +469,32 @@
         r4 (or (get corners 3) 0)]
     (h/call internal-module "_set_shape_corners" r1 r2 r3 r4)))
 
+
+(defn- translate-shadow-style
```

Comment:
Shouldn't we add some documentation about this in render-wasm/docs/serialization.md?